Source/ACE/Network/GameMessages/Messages/GameMessageMotion.cs

```diff
@@ -46,15 +53,18 @@ public GameMessageMotion(Entity.WorldObject animationTarget, Session session, Mo
             // in AC Log View it is reversed.
             // TODO: reseach the correct order.   We can probably get away with either for now.
             Writer.Write((ushort)type); // movement_type
-            Writer.Write((byte)activity); // autonomous flag - 1 or 0.   I think this is set if you have are holding the run key or some other autonomous movement            
+            Writer.Write((byte)activity); // autonomous flag - 1 or 0.   I think this is set if you have are holding the run key or some other autonomous movement
             Writer.Write((byte)flags); // these can be or and has sticky object | is long jump mode |
             Writer.Write((ushort)stance); // called command in the client
 
         }
 
         private void WriteAnimations(WorldObject animationTarget, List<MotionItem> items)
         {
-            if (animationTarget == null) throw new ArgumentNullException(nameof(animationTarget));
+            if (animationTarget == null)
+            {
+                Log("We have a null for animationTarget - that is wrong, wrong wrong.");
```

Comment:
Oops.   Can't I am rusty can you 😛   Will fix as soon as I get home.   Thanks for reviewing for me